application.ts

```diff
@@ -16,8 +16,7 @@ import type {
   ServerResponse,
   ServeTls,
 } from "./types.d.ts";
-import { reset } from "https://deno.land/std@0.71.0/fmt/colors.ts";
-import { resolve } from "https://deno.land/std@0.71.0/path/win32.ts";
```

Comment:
These were unused, and direct imports instead of through `deps.ts`

Reply:
yeah, I saw those strangely the other day... must of been auto-imports that leaked in... 😭   Thank for cleaning that up.